### DISCUSSION TITLE
Is Obsidian Git safe to use in a team settings?

### DISCUSSION THREAD

**Discussion by jmfayard** [Vinzent03/obsidian-git#312] [Q&A]
Bonjour

## tl:dr

is Obsidian Git safe to use with multiple developers, especially with things like auto-fetch, auto-commit, auto-pull, auto-messages?

## Context:

My relatively small team (4 backend developers, 5 front-end developers) is currently relying on Confluence for documentation.
Interestingly things are not really well documented, and the documentation that is there is often not found or not used.

I was thinking about something like Notion, which works well in a team and would already be a step forward.

But then I remember I personally love Obsidian + git.
I thought I could use a GitHub wiki to host the wiki so people wouldn't even need Obsidian installed when they don't have it.
That's possible by using an URL like `git clone https://github.com/denolehov/obsidian-git.wiki.git`

## Questions

- are people already using Obsidian Git in a team settings?
- how well does that work? Any pitfalls do be aware of?
- which settings do you recommend?

**Comment by Vinzent03** [comment]
Since this is just typical Git, I think it works really well for teams. BUT the work needs to be async, like you can't edit a file live with your teammate. You have to commit and push the changes. It's not instantly synced like on Notion. I suggest frequent commits to prevent merge conflicts. I don't know your work to know if automatic backups are useful or instead commit manually.

**Reply by quacau** [reply]
I recently ask this question on Software Recommendations Stack Exchange for syncing in collaboration for non-technical team for Obsidian. I hope it will be helpful: [File synchronization solution for controlling type of sync for subfolders and devices?](https://softwarerecs.stackexchange.com/q/84233/6389)
